TMS.NET06.BookingService.Spa/ClientApp/src/components/Userinfo.js

```diff
@@ -34,6 +34,7 @@ export class Userinfo extends Component {
             })
         });
         const data = await response.json();
+        console.log(data);
```

Comment:
Please get rid of console.log in the code, it usually is used only for dev purposes